pipeline-examples/github-org-plugin/access-repo-information.groovy

```diff
@@ -2,7 +2,7 @@
 
     // github-organization-plugin jobs are named as 'org/repo/branch'
     tokens = "${env.JOB_NAME}".tokenize('/')
-    org = tokens[0]
-    repo = tokens[1]
-    branch = tokens[2]
+    org = tokens[tokens.size()-3]
```

Comment:
@kmadel Would you mind adding somments to this script on why it's important to use `tokens.size()` here?